src/App.vue

```diff
@@ -1318,13 +1323,21 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   /** Fetches authorizations and verifies roles. */
   private async checkAuth (id: string): Promise<any> {
     // NB: will throw if API error
-    const authRoles = await AuthServices.fetchAuthorizations(id)
+    const authorizations = await AuthServices.fetchAuthorizations(id)
+    console.log('*** authorizations =', authorizations)
+
+    const authRoles = authorizations.roles || []
 
-    // verify that array has at least one role
-    // NB: roles array may contain 'view', 'edit', 'staff' or nothing
-    if (!Array.isArray(authRoles) || authRoles.length < 1) {
+    if (!Array.isArray(authRoles)) {
       throw new Error('Invalid auth roles')
     }
+
+    this.setAuthorizations(authorizations)
+
+    // verify that array has "view" or "staff" roles
+    if (!authRoles.find(r => r === 'view' || r === 'staff')) {
+      throw new Error('Missing required role')
+    }
```

Comment:
I added this because we previously only checked for at least 1 role.

This check may change as I work through roles and permissions with SBC Connect Team.

Reply:
Update: it appears that all account types have either role "view" or "staff", so the above should be good.

Reply:
I did this in a separate PR: https://github.com/bcgov/business-create-ui/pull/785

This conversation is out of date - resolving it.